tracing/README.md

```diff
@@ -319,3 +319,19 @@ grpcSrv := grpc.NewServer(
 	grpc.StreamInterceptor(otelgrpc.StreamServerInterceptor()),
 )
 ```
+
+### Config Options
+Possible environment config exporter config options when using `tracking.InitTracing()`.
+
+#### Jaeger
+* `OTEL_EXPORTER_JAEGER_PROTOCOL`
```

Comment:
It would be helpful to explain the possible uses of this setting.  I'll take a stab:

```markdown
* `OTEL_EXPORTER_JAEGER_PROTOCOL`
Possible values:

   * `udp/thrift.compact` (default): Export traces via UDP datagrams.  Best used when Jaeger Agent is accessible via loopback interface.  May also provide `OTEL_EXPORTER_JAEGER_AGENT_HOST`/`OTEL_EXPORTER_JAEGER_AGENT_PORT`, which default to `localhost`/`6831`. 
   * `udp/thrift.binary`: Alternative protocol to the more commonly used `udp/thrift.compact`.  May also provide `OTEL_EXPORTER_JAEGER_AGENT_HOST`/`OTEL_EXPORTER_JAEGER_AGENT_PORT`, which default to `localhost`/`6832`. 
   * `http/thrift.compact`: Export traces via HTTP packets.  Best used when Jaeger Agent cannot be deployed or is inaccessible via loopback interface.  This setting sends traces directly to Jaeger's collector port.  May also provide `OTEL_EXPORTER_JAEGER_ENDPOINT`, which defaults to `http://localhost:14268/api/traces`.

```